RCSwitch.cpp

```diff
@@ -45,6 +46,7 @@ unsigned long RCSwitch::nReceivedValue = 0;
 unsigned int RCSwitch::nReceivedBitlength = 0;
 unsigned int RCSwitch::nReceivedDelay = 0;
 unsigned int RCSwitch::nReceivedProtocol = 0;
+unsigned int RCSwitch::enabledProtocols = B00000111;
```

Comment:
The name of this member variable is misleading -- a better one would be `enabledReceiveProtocols` or `enabledReceptionProtocols` or so